### DISCUSSION TITLE
See the size of folders

### DISCUSSION THREAD

**Discussion by ircserkan** [valinet/ExplorerPatcher#2201] [Ideas]
Hello
Is it possible to show the size of the folders? When I make the folder view detailed in Windows Explorer, there is a size section, but it does not show the size of the folder. Could such a feature be added to Explorer Patch?

**Comment by pyrates999** [comment]
Not if windows itself doesn't return that information.

**Comment by ircserkan** [comment]
When you hover over the folder with the mouse, it gives you the file size. However, it does not show in the File Size column. If it is possible to ensure this, it will be very good to do it.

**Comment by pyrates999** [comment]
The source code is available, so you're welcome to try to add that feature.

But I think it is not enabled due to performance reasons, otherwise windows would be constantly accessing the drives to get their size data for every directory

https://answers.microsoft.com/en-us/windows/forum/all/windows-file-explorer-not-showing-folder-size/7b31df8a-3fe7-41d2-97c9-d5cfb6c93cf1

It is also limited to paths of 260 characters in length.
